### PR TITLE
Update medicine.yml

### PR DESCRIPTION
**Pull request by Suraru**
Merges the med doc by floof.
At least one original change by XavierSomething (will update when other PRs are found)

https://github.com/Fansana/floofstation1/pull/222

### CODE REVIEW

**Comment by Aidenkrz** [comment]
The `Fixes Leporazine Temperature Shock` should be the only thing in this PR, rebalancing chems should be in a seperate PR.

**Review by FoxxoTrystan — DISMISSED** [review]
Remove Floof Comments, and mostly will need review from others maintainers regarding durk opinion.

**Review by VMSolidus — APPROVED** [review]

**Review by sleepyyapril — APPROVED** [review]

**Pull request merged**
